tests/phpunit/includes/factory/unittest-factory-for-language.php

```diff
@@ -90,7 +90,12 @@ public function create_object( $args ) {
 			return $language;
 		}
 
-		return $language->term_id;
+		// Backward compatibility with Polylang < 3.8.
+		if ( ! $language instanceof PLL_Language ) {
+			$language = $this->pll_model->languages->get( $args['locale'] );
```

Comment:
I propose to use the slug when possible:
```suggestion
			$language = $this->pll_model->languages->get( $args['slug'] ?? $args['locale'] );
```
to handle correctly cases where we would add two languages with the same locale.